src/test/java/com/synopsys/integration/jira/common/JiraTestConstants.java

```diff
@@ -0,0 +1,23 @@
+package com.synopsys.integration.jira.common;
+
+public final class JiraTestConstants {
+
```

Comment:
Gross

Reply:
You don't like this? I could name it better but I like having all the Test Environment Variables in a single place so I know everything I would need to add if I were new to the project.

Reply:
I'm talking about line 4. The whitespace.